ui/app/components/app/modals/deposit-ether-modal/deposit-ether-modal.component.js

```diff
@@ -133,7 +133,7 @@ export default class DepositEtherModal extends Component {
                 });
                 toWyre(address);
               },
-              hide: isTestNetwork,
+              hide: isTestnet,
```

Comment:
This should be checking whether the current network is Mainnet. This would still show the Wyre section for custom networks.

```suggestion
              hide: !isMainnet,
```